### ISSUE TITLE
seems like not working

### ISSUE THREAD

**Issue by mrdiega** [hg-fortis/Force-AllTalk#1]
This error appears when starting the server and every round
```
"McCoy<1><BOT><CT>" OnPreResetRound => CTMDBG, team 3  will switch 0 265.44
"diega.WinWin^^<2><TERRORIST>" OnPreResetRound => CTMDBG, team 2  will switch 0 265.44
01:21:21 [EROR] (cssharp:Core) Error invoking callback
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.InvalidOperationException: ConVar is a Bool but you are trying to get a System.Int32 value.
   at CounterStrikeSharp.API.Modules.Cvars.ConVar.GetPrimitiveValue[T]() in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Modules/Cvars/ConVar.cs:line 40
   at CounterStrikeSharp.API.Modules.Cvars.ConVar.SetValue[T](T value) in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Modules/Cvars/ConVar.cs:line 93
   at Force_AllTalk.ForceAllTalk.OnRoundStart(EventRoundStart event, GameEventInfo info) in E:\Code\repos\ForceAllTalk\Force AllTalk\Force AllTalk\ForceAllTalk.cs:line 49
   at InvokeStub_GameEventHandler`1.Invoke(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   --- End of inner exception stack trace ---
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at CounterStrikeSharp.API.Core.FunctionReference.<>c__DisplayClass3_0.<.ctor>b__0(fxScriptContext* context) in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Core/FunctionReference.cs:line 82
```
It looks like the plugin is not working
![full alltalk](https://github.com/hg-fortis/Force-AllTalk/assets/79888956/407cf7bb-1a3a-4ab2-921a-9150ca0c3d2f)

counterstrikesharp-build-84-linux-6b0912d

**Comment by hg-fortis** [comment]
Simple oversight by me, will fix when I get home tonight

Thanks for reporting it!

**Comment by fortiiis** [comment]
Fixed as of v1.1
